Makefile

```diff
@@ -21,6 +21,8 @@ bdist-wheel:
 
 bundle-docs-clean:
 	rm -rf docs/pyteal.docset
+	rm -rf docs/_build
```

Comment:
Needed this change in order to guarantee getting an error when I ought to be getting an error when running `make bundle-docs`

Reply:
👍